src/components/hero-header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import cn from 'classnames';
+
+import '../styles/components/header.scss';
+
+type HeroHeaderProps = {
+  title: JSX.Element;
+  children: JSX.Element;
+  footer?: JSX.Element;
+  className?: string;
+};
+
+const HeroHeader: React.FC<HeroHeaderProps> = ({
+  title = '',
+  children,
+  footer,
+  className,
+  ...rest
```

Comment:
After checking usage I'd call it `props` instead of `rest` as it's more consistent with the rest of the codebase